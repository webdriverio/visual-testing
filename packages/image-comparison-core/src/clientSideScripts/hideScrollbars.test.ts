// @vitest-environment jsdom

import { describe, it, expect } from 'vitest'
import hideScrollBars from './hideScrollbars.js'

describe('hideScrollBars', () => {
    it('should be able to hide and show the scrollbars', () => {
        expect(document.body.style.overflow).toMatchSnapshot()

        hideScrollBars(true)

        expect(document.body.style.overflow).toMatchSnapshot()

        hideScrollBars(false)

        expect(document.body.style.overflow).toMatchSnapshot()
    })
})
