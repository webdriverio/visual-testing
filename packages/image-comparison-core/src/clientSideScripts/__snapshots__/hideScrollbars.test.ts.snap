// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`hideScrollBars > should be able to hide and show the scrollbars 1`] = `""`;

exports[`hideScrollBars > should be able to hide and show the scrollbars 2`] = `"hidden"`;

exports[`hideScrollBars > should be able to hide and show the scrollbars 3`] = `""`;
